tools/old/softirqs.py

```diff
@@ -115,11 +166,18 @@
         "rcu_process_callbacks", "run_rebalance_domains", "tasklet_action",
         "tasklet_hi_action", "run_timer_softirq", "net_tx_action",
         "net_rx_action"):
-    b.attach_kprobe(event=softirqfunc, fn_name="trace_start")
-    b.attach_kretprobe(event=softirqfunc, fn_name="trace_completion")
+    if args.CPUidx:
```

Comment:
Again, there's no need for this duplication if we give the C functions the same names in both cases. I don't know why not.